runner/test/runner/pipeline_test.clj

```diff
@@ -305,9 +305,16 @@
                            run-info (xt/entity (xt/db db) (keyword (str "bob.pipeline.run/" result)))
                            statuses (->> history
                                          (map ::xt/doc)
-                                         (map :status))]
-                       (is (= [:passed :running :initializing]
-                              statuses))
+                                         (map :status)
+                                         (into #{}))]
+                       (is (inst? (:initiated-at run-info)))
```

Comment:
I don't understand why this tests is failing, I does has the `initializing` status in the history which is added at the same time 😕

Reply:
sometimes is this line or the `395` but both is related to the :initiated-at